### ISSUE TITLE
Out of date website significantly undersells you

### ISSUE THREAD

**Issue by StoneCypher** [ultralight-ux/ultralight#492]
Sorry, I looked for a website repo to open a PR against, but there isn't one visible

Your README says you do `Windows, macOS, Linux, Xbox, PS4, and PS5 (and ARM64 coming in 1.4)`

Your website landing page says `Currently available for Windows, macOS, Linux, and Xbox.`

Your website FAQ on the platform section says `Ultralight supports most major desktop PC platforms. Specifically, Windows (7+, x64), macOS (Sierra or later), and Linux (Ubuntu/Debian 9.5+, x64).`

**Comment by adamjs** [comment]
Hello! Good catch, we're launching a new website shortly with all of that information (console support and new features). 😀

**Comment by JailbreakPapa** [comment]
Yeah, also agree. would be nice to see the new website update soon!
